OrbitGl/Batcher.cpp

```diff
@@ -103,6 +103,28 @@ void Batcher::AddTriangle(const Triangle& triangle, const Color& color, const Co
   user_data_.push_back(std::move(user_data));
 }
 
+void Batcher::AddCircle(const float& z, const float& radius, const float& x_pos, const float& y_pos,
+                        const Color& color) {
+  std::vector<Vec2> circle_points_scaled_by_radius;
+  std::transform(circle_points.begin(), circle_points.end(),
+                 std::back_inserter(circle_points_scaled_by_radius),
+                 [&radius](const Vec2& p) -> Vec2 { return Vec2(p[0] * radius, p[1] * radius); });
```

Comment:
for(auto& point : circle_points) {
  circle_points_scaled_by_radius.emplace_back(radius*point);
}

Reply:
Thank you!